analysis_options.yaml

```diff
@@ -11,7 +11,9 @@ include: package:flutter_lints/flutter.yaml
 
 analyzer:
   exclude: [lib/**.freezed.dart]
```

Comment:
Please add `lib/**/*.g.dart` to the array as well to prevent the analyzer from reporting issues in .g.dart files across the entire project.

Reply:
Let's focus on code review rather than pipeline. Someone can do a PR of the same later.